docs/core/docker/intro-net-docker.md

```diff
@@ -1,191 +1,64 @@
-
-The samples use .NET Core 2.0. They use Docker [multi-stage build](https://github.com/dotnet/announcements/issues/18) and [multi-arch tags](https://github.com/dotnet/announcements/issues/14) where appropriate.
-
-* [.NET Core images on DockerHub](https://hub.docker.com/r/microsoft/dotnet/)
-
-* [Dockerize a .NET Core application](https://docs.docker.com/engine/examples/dotnetcore/)
-
-* This .NET Core Docker sample demonstrates how to [use Docker in your .NET Core development process](https://github.com/dotnet/dotnet-docker-samples/tree/master/dotnetapp-dev). The sample works with both Linux and Windows containers.
-
-* This .NET Core Docker sample demonstrates a best practice pattern for [building Docker images for .NET Core apps for production.](https://github.com/dotnet/dotnet-docker-samples/tree/master/dotnetapp-prod) The sample works with both Linux and Windows containers.
-
-* This [.NET Core Docker sample](https://github.com/dotnet/dotnet-docker-samples/tree/master/dotnetapp-selfcontained) demonstrates a best practice pattern for building Docker images for [self-contained .NET Core applications](../deploying/index.md). Used for the smallest production container without a benefit from [sharing base images between containers](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/). However, lower Docker layers could be shared.
-
-#### ARM32 / Raspberry Pi
-
-* [.NET Core Runtime ARM32 builds announcement](https://github.com/dotnet/announcements/issues/29)
-
-* [ARM32 / Raspberry Pi .NET Core images on DockerHub](https://hub.docker.com/r/microsoft/dotnet/)
+### Images
 
-* [ARM32 / Raspberry Pi .NET Core Docker Samples on GitHub](https://github.com/dotnet/dotnet-docker-samples#arm32--raspberry-pi)
+An image is a read-only template with instructions on how to create a container. Much the time an image is based on another image, but with some customization. For example, when you create an image for your application, you would base it on an existing image that already contains the .NET Core runtime.
 
-#### .NET Framework
+### Containers
 
-* [.NET Framework images on DockerHub](https://hub.docker.com/r/microsoft/dotnet-framework/)
+A container is a runnable instance of an image. As you build your image, you deploy your application and dependencies. Then, multiple containers can be instantiated, each isolated from one another. Each container instance has its own filesystem, memory, network interface.
 
-This repo contain samples that demonstrate various .NET Framework Docker configurations. You can use these images as the basis of your own Docker images.
+### Registries
 
-**.NET Framework 4.7**
+Container registries are repositories of container images. Docker has a public registry hosted at the [Docker Hub](https://hub.docker.com/) that you can use. .NET Core related images are listed at the Docker Hub.
```

Comment:
I'd add that you retrieve images from repositories (or you can base your images on images in a repository).